docs/examples/notebooks/learn/TestStatistics.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>need to explain why one chooses <code>[1,1]</code> here. also explain that you're doing 1000 toys.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scikit-hep/pyhf/pull/1014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>